Week3/homework/7-step3.js

```diff
@@ -21,5 +21,6 @@ f2(y);
 
 console.log(y);
 
-// Add your explanation as a comment here
-
+/*
+f1() is a pointer and f2() is a reference
+*/
```

Comment:
Nope.  f1() is passed an atomic value (num, boolean etc.) which are passed by value, meaning that they are copied.  The second f2() is a compound object and is passed by reference.